src/content/docs/en/pages/devtools/azion-edge-runtime/compatibility/frameworks.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Frameworks compatibility
+description: Compatibility between Azion and web frameworks through Vulcan.
+namespace: documentation_products_runtime_frameworks
+meta_tags: web frameworks, jamstack
+permalink: /documentation/products/devtools/frameworks-compatibility/
+menu_namespace: runtimeMenu
+
+---
+
+import Badge from '~/components/Badge.astro';
+
+
+## [Vulcan](https://github.com/aziontech/vulcan)
+
+[Vulcan](https://github.com/aziontech/vulcan) is Azion's framework adapter built to run on our Edge Runtime. It's an evolving project aimed at supporting various JavaScript frameworks. The project is open source and encourages community contributions.
+
+Azion provides different ways to get started and work with web frameworks: 
+
+- [Azion CLI](/en/documentation/products/azion-cli/overview/)
+- [Real-Time Manager](/en/documentation/products/start-with-a-template/)
```

Comment:
suggestion: /en/documentation/products/marketplace/templates/#acceleration-of-framework-adoption